crates/extrinsics/src/contract_artifacts.rs

```diff
@@ -142,6 +143,20 @@ impl ContractArtifacts {
         })
     }
 
+    /// Get the deserialized [`InkProject`] metadata.
+    ///
+    /// ## Errors
+    /// - No contract metadata could be found.
+    /// - Invalid contract metadata.
+    pub fn ink_project_metadata(&self) -> Result<InkProject> {
```

Comment:
I think we should consider updating the 'scheme' command to use 'contract_artifacts'. There is some code duplication.

Reply:
I hadn't even looked at the `schema` command, but just looked and you are absolutely right it could/should use `ContractArtifacts`. Should be a separate PR though.

Reply:
Yh, it makes sense to reuse this method for schema operations